src/Pass/ImgTagTransformPass.php

```diff
@@ -110,6 +201,19 @@ protected function getImageWidthHeight($src)
 
         // Try obtaining image size without having to download the whole image
         $size = $this->fastimage->getImageSize($img_url);
```

Comment:
Check with @cerber717.  There's a trivial way to do this from most of our image pathnames now.   We should leverage that as well when it's available.